server/src/main/java/org/elasticsearch/index/mapper/IndexFieldMapper.java

```diff
@@ -88,6 +87,11 @@ public List<Object> fetchValues(SourceLookup lookup, List<Object> ignoredValues)
                 }
             };
         }
+
+        @Override
+        public boolean eagerGlobalOrdinals() {
+            return false;
```

Comment:
Is this related?

Reply:
The parent value is false as well, so it's not really necessary, but it clarifies that this won't be called during index warming and so we don't need the fully qualified index name and can happily use an empty fielddatacontext at that point.

Reply:
Maybe a comment and make it `final`?

Reply:
It's on a final class so checkstyle whinges if the method is marked as final 🤷